app/lib/search/search_service.dart

```diff
@@ -273,6 +281,7 @@ class SearchQuery {
     final String q = uri.queryParameters['q'];
     final String platform =
         uri.queryParameters['platform'] ?? uri.queryParameters['platforms'];
+    final owners = uri.queryParametersAll['owners'];
```

Comment:
The semantics is that you want results filtered by "is owned by one of these", right?

Hence, I suggest the query parameter name `owned_by_one_of`, so url would be `../?q=...&owned_by_one_of=jonasfj@google.com,dart.dev`.

Then we just do `(uri.queryParameter['owned_by_one_of'] ?? '').split(',')` or something like that.

Reply:
We could do that too, but this is a backend URL, which we don't expose to the users, so it could also use the already built-in multi-value URL parsing. At some point we probably need to have a proper JSON API for this too.